Luke7787/ImmaculateInventors
# Feature/items

Summary: 
**- Developed entry grid, table grows downward by increments of 4
- Entries holds (name, quantity, image (JPG), delete button)
- Implemented delete button
- Implemented, Button "+ New Item"
- Changed Front : 'Roboto', sans-serif
- Converts all image to same size (110 x 110 pixle)**

Link to any specs used: **[https://www.figma.com/file/XP0UJUNIP8dEv6lqgWMQEg/UI-Design?type=design&node-id=0-1&mode=design&t=jlNW4Ja6D0Ypa3ts-0](url)**

How to test: 
**- npm start
- Press delete on entries you want to delete - Tested ✓
- Make sure the format doesn't break format (Long Names) - Tested ✓
- Handle difference image size Tested ✓**

Notes: **Button "+ New Item" is not implemented yet**
